### PR TITLE
Add the meetup agenda of 1002

### PR DESCRIPTION
**Pull request by ThreeMonth03**
The 1002 agenda  is same as the agenda in 0925

### CODE REVIEW

**Comment by ThreeMonth03** [comment]
@YenPeiChen07 @meng5113 Could you please review this pr? Thank you.

File: content/pages/meetup/2024/1002-nycu.rst [hunk 1]
```diff
@@ -0,0 +1,141 @@
+========================================
+Meetup 2024 September 25th in NYCU
```
Comment (YenPeiChen07) [inline content/pages/meetup/2024/1002-nycu.rst:2]: This title needs to be updated to **October 2nd**.

**Review by YenPeiChen07 — CHANGES_REQUESTED** [review]
Everything looks great. Just the event tile needs to be updated.

File: content/pages/meetup/2024/1002-nycu.rst (hunk 1 above)
Comment (ThreeMonth03) [reply content/pages/meetup/2024/1002-nycu.rst]: FIx.

**Review by YenPeiChen07 — APPROVED** [review]

**Comment by ThreeMonth03** [comment]
Thanks, I decide to merge this pull request now.

**Pull request merged**